internal/k8s/controller.go

```diff
@@ -557,12 +583,12 @@ func (c *AviController) SetupEventHandlers(k8sinfo K8sinformers) {
 			namespace, _, _ := cache.SplitMetaNamespaceKey(utils.ObjKey(svc))
 			if isSvcLb && !lib.GetLayer7Only() {
 				if !utils.IsServiceNSValid(namespace) {
-					utils.AviLog.Debugf("L4 Service delete event: Namespace: %s didn't qualify filter. Not deleting service.", namespace)
```

Comment:
We usually don't ahem Info logs here, else there may be too many logs.

Reply:
@monotosh-avi :True. Changes has been made to make debugging easier when namespace feature enabled in customer env as well as in our integration pipeline. If you remember, 2 week back, there was issue in multinamespace oshift integration test cases where we were not able to determine whether namespace filtering has been applied to namespace or not. To take care such scenarios, changed this.

Reply:
I think we would end up getting too many logs messages. May be we can run the UTs in Debug mode ?